internal/build/gen_mk.sh

```
@@ -106,7 +106,7 @@ EOT
 
 gen_make_targets() {
 	local cmd="$1" name="$2" dir="$3" mod="$4" deps="$5"
-	local call= callu= callx=
+	local call= callu=
```

Shellcheck

[warning] In POSIX sh, 'local' is undefined.

(SC3043)